packages/@okta/vuepress-site/docs/release-notes/2022/index.md

```diff
@@ -4,6 +4,16 @@ title: Okta API Products release notes 2022
 
 ## June
 
+### Weekly release 2022.06.1
+
+| Change | Expected in Preview Orgs |
+|--------------------------------------------------------------------------|--------------------------|
+| [Bug fixed in 2022.06.1](#bug-fixed-in-2022-06-1) | June 15, 2022 |
+
+#### Bug fixed in 2022.06.1
+
+Resource Set operations performed with Okta Resource Names (ORNs) that used capital letters returned an HTTP 500 Internal Server Error. (OKTA-501910)
```

Comment:
Should there be a reference to this api: [https://developer.okta.com/docs/reference/api/roles/#resource-set-operations](https://developer.okta.com/docs/reference/api/roles/#resource-set-operations)? Just wondering.

Reply:
My understanding from my 1:1 with Janet yesterday is we should follow what infodev does (no links in bugs) until we clarify what devdocs should do.